src/networking.c

```diff
@@ -4100,7 +4100,8 @@ void clientTrackingInfoCommand(client *c) {
 
     /* Prefixes */
     addReplyBulkCString(c, "prefixes");
-    if (c->pubsub_data->client_tracking_prefixes) {
+    if (c->flag.tracking) {
+        serverAssert(c->pubsub_data && c->pubsub_data->client_tracking_prefixes);
```

Comment:
I guess your original fix was better. :cry:

`c->pubsub_data->client_tracking_prefixes` doesn't always exist when tracking is on.

Maybe this works?

```suggestion
    if (c->flag.tracking && c->pubsub_data->client_tracking_prefixes) {
```